isamples_metadata/GEOMETransformer.py

```diff
@@ -4,12 +4,34 @@
     Transformer,
 )
 
+TISSUE_ENTITY = "Tissue"
+
+
+# Forward declarations appear to be the cleanest way to manage the circular dependency between these two classes
```

Comment:
@datadavev -- curious if this looks alright to you.  In C we handle this with forward declarations, and this appears to be the Python equivalent.  I saw some other suggestions on the internet that involved goofy class interpolation through strings, but I have no interest in writing, reading, or maintaining such crimes if it can be avoided.

Reply:
another approach is to pass the `GEOMEChildTransformer` class as a parameter to `GEOMETransformer.__init__()`, not sure it offers any benefit though. Like:
```
    def __init__(self, source_record: typing.Dict, child_class):
        super().__init__(source_record)
        self._child_transformers = []
        children = self._get_children()
        for child_record in children:
            entity = child_record.get("entity")
            if entity == TISSUE_ENTITY:
                self._child_transformers.append(
                    child_class(source_record, child_record)
                )
```

Reply:
Yeah, I suppose that would work too.  Do you feel strongly about it?  I don't like the idea of making callers need to know which child class to instantiate, though.

Reply:
I think it's cleaner with the forward declarations - easier to read and as you say, which child class to pass in?